install.sh

```diff
@@ -370,6 +370,10 @@ setup_tmp() {
     set +o errexit
```

Comment:
Simplify:
```sh
[ "$_exit_code" -eq 0 ] || error "Install script exited with code $_exit_code"
```
Moreover, I think this should be moved right after `_exit_code=$?`
Note: Use `error` log instead of `fatal`

Reply:
I've tried to stay with your coding style actually, since you use if [ ]; then ; fi semantic in the script already